src/Crypto/RNCryptor/Types.hs

```diff
@@ -99,8 +113,9 @@ renderRNCryptorHeader RNCryptorHeader{..} =
 -- A convenient datatype to avoid carrying around the AES cypher,
 -- the encrypted key and so on and so forth.
 data RNCryptorContext = RNCryptorContext {
-        ctxHeader :: RNCryptorHeader
-      , ctxCipher :: AES
+        ctxHeader  :: RNCryptorHeader
+      , ctxCipher  :: AES128
```

Comment:
Are we sure the change for the type of `ctxCipher` from `AES` to `AES128` is the correct thing? The data format for RNCryptor talks about `AES256`:

https://github.com/RNCryptor/RNCryptor-Spec/blob/master/RNCryptor-Spec-v3.md#password-based-encryption-abstract-language